projects/storefrontlib/src/layout/layout.module.ts

```diff
@@ -7,7 +7,7 @@ import { LayoutConfig } from './config/layout-config';
 const layoutModules = [OutletRefModule];
 
 @NgModule({
-  imports: [...layoutModules, SkipLinkModule],
+  imports: [...layoutModules, SkipLinkModule, OutletRefModule],
```

Comment:
with that change, you should remove the `layoutModules`, as it's obsolete now I believe